src/next/azext_next/custom.py

```diff
@@ -190,15 +190,26 @@ def _execute_nx_cmd(cmd, nx_cmd, nx_param, catch_exception=False):
         if param in store_true_params:
             args.append(param)
         else:
-            print("Please input " + Fore.LIGHTBLUE_EX + param + Fore.RESET + ":", end='')
-            value = input()
+            print("Please input " + Fore.LIGHTBLUE_EX + param + Fore.RESET + ": ", end='')
```

Comment:
Please consider using the new color framework.

Reply:
OK~